Makefile

```diff
@@ -27,7 +27,7 @@ test: test/test test/piggie
 	test/piggie
 	test/test dump `pidof piggie` image
 	test/test restore image
-	killall -9 piggie || :
+	pkill -9 piggie || :
```

Comment:
I am not very sure to be honest but I think it might be related to some of the "known bugs" mentioned in the man page:

```
Killing by file only works for executables that are kept open during
execution, i.e. impure executables can't be killed this way.

If processes change their name, killall may not be able to match them correctly.
```